notebooks/code_samples/custom_tests/integrate-external_test_providers.ipynb

```diff
@@ -154,11 +211,11 @@
       ]
     },
     {
-      "attachments": {},
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "### Register external test providers\n",
+        "<a id='toc4_2_'></a>\n\n",
+    "### Register external test providers\n",
         "\n",
         "We will now instantiate and register test provider classes that include the tests we included in the template.\n",
```

Comment:
Good suggestion, though a wise editor once told me to avoid text in brackets and use present tense. 😁 Text now reads:

`We now instantiate and register test provider classes that include the tests we included in the template. Instantiating creates a new context based on a pre-existing model or scheme.\n`